priv/repo/migrations/20180830224557_create_login.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Ret.Repo.Migrations.CreateLogin do
+  use Ecto.Migration
+
+  def change do
+    create table(:logins, prefix: "ret0", primary_key: false) do
+      add(:login_id, :bigint, default: fragment("ret0.next_id()"), primary_key: true)
+      add(:identifier_hash, :string, null: false)
+      add(:account_id, references(:accounts, column: :account_id, on_delete: :delete_all))
+
+      timestamps()
+    end
+
+    create(index(:logins, [:identifier_hash], unique: true))
+    create(index(:logins, [:account_id]))
```

Comment:
I'm actually not sure -- this system might eventually support more than one login identifier for an account. I'm still undecided about the sanity of separating these tables but that is one reason having them separate would be a good idea. I think you're right that we can add the index now though since we're not supporting it.